# /** ------ 交通灯信息 ------ */
int32                    u4Id
int32                    u4CtrlId
uint32                   u4NoPhases
uint32                   u4DataSize
float32                  u4State
float32                  u4CycleTime
traffic_light_phase[]    phases